Sources/swift-parsing-benchmark/JSON.swift

```diff
@@ -174,10 +176,11 @@ let jsonSuite = BenchmarkSuite(name: "JSON") { suite in
     }
   )
 
+  let dataInput = Data(input.utf8)
   var objectOutput: Any!
   suite.benchmark(
     name: "JSONSerialization",
-    run: { objectOutput = try JSONSerialization.jsonObject(with: Data(input.utf8), options: []) },
+    run: { objectOutput = try JSONSerialization.jsonObject(with: dataInput, options: []) },
```

Comment:
The benchmark wasn't "fair" to JSONSerialization since it was insuring the `Data.init` cost.